charts/kubefed/charts/controllermanager/templates/deployments.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       containers:
       - command:
         - /hyperfed/controller-manager
+        - "--v=2"
```

Comment:
add a command property to the values.yaml for the controller so this verbosity level can be re-defined.